src/middlewared/middlewared/plugins/nfs.py

```diff
@@ -21,20 +24,20 @@ def nfs_compress(self, nfs):
         return nfs
 
     @accepts(Dict(
-        'nfs_update',
-        Int('servers', validators=[Range(min=1, max=256)]),
-        Bool('udp'),
-        Bool('allow_nonroot'),
-        Bool('v4'),
-        Bool('v4_v3owner'),
-        Bool('v4_krb'),
-        List('bindip', items=[IPAddr('ip')]),
-        Int('mountd_port', required=False, validators=[Range(min=1, max=65535)]),
-        Int('rpcstatd_port', required=False, validators=[Range(min=1, max=65535)]),
-        Int('rpclockd_port', required=False, validators=[Range(min=1, max=65535)]),
-        Bool('userd_manage_gids'),
-        Bool('mountd_log'),
-        Bool('statd_lockd_log'),
+        "nfs_update",
+        Int("servers", validators=[Range(min=1, max=256)]),
+        Bool("udp"),
+        Bool("allow_nonroot"),
+        Bool("v4"),
+        Bool("v4_v3owner"),
+        Bool("v4_krb"),
+        List("bindip", items=[IPAddr("ip")]),
+        Int("mountd_port", required=False, validators=[Range(min=1, max=65535)]),
+        Int("rpcstatd_port", required=False, validators=[Range(min=1, max=65535)]),
+        Int("rpclockd_port", required=False, validators=[Range(min=1, max=65535)]),
+        Bool("userd_manage_gids"),
+        Bool("mountd_log"),
+        Bool("statd_lockd_log"),
```

Comment:
I guess we might need to discuss single quotes versus quotes.

My OCD twitches when these are changed without any reason : D

Is there any reason you use double quotes? Is that part of PEP8?

Reply:
> This PEP does not make a recommendation for this. Pick a rule and stick to it.

I prefer double quotes because they are also used in C code, JSON literals, they just look nice to me, also

> For triple-quoted strings, always use double quote characters to be consistent with the docstring convention in PEP 257.

In this particular case I did find&replace on the whole document for reason I can't remember

Reply:
Right.

I just like single quote because its one press key instead of two.

So do we need some sort of document to describe code standard for middlewared here or should we contain ourselves to not change code just for the sake of change and everyone does what it feels right in its own boundaries? Thoughts?